app/facades/food_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodFacade
+  def self.foods_by_ingredient(ingredient)
+    json = FoodService.foods_by_ingredient(ingredient)
+    json[:foods].map do |food_data|
+      Food.new(food_data)
+    end
+  end
```

Comment:
This all looks good! The foods service does come with a totalHits attribute that will give the number of how many results came from that search. I belive you could also grab this from your json result and sent that back to your controller to render to your view instead of calling `count` in your view.